### PR TITLE
Feature/587

### PR DESCRIPTION
**Pull request by Haytham1337**
Write functionallity for adding sizes to model
image

### CODE REVIEW

File: src/components/forms/model-form/model-form.js [hunk 1]
```diff
@@ -129,42 +123,43 @@ const ModelForm = ({ model, id, isEdit }) => {
...
-  }
+  };
 
   const inputs = [
-    { label: name, name: 'name' },
-    { label: description, name: 'description', isEditor:true },
+    { label: name, name: labelsEn.name },
+    { label: description, name: labelsEn.description, isEditor: true }
   ];
   const inputOptions = {
```
Comment (AndriyLymych) [inline src/components/forms/model-form/model-form.js:164]: put it into utils

File: src/components/forms/model-form/model-form.js [hunk 2]
```diff
@@ -63,38 +66,28 @@ const ModelForm = ({ model, id, isEdit }) => {
...
   const [category, setCategory] = useState(model.category._id || '');
 
   const handleCategory = (event) => {
     values.category = event.target.value;
     setCategory(event.target.value);
   };
-
+  const onTagsChange = (event, value) => {
+    const sizeIdlist = value.map((size) => size._id);
+    values.sizes = sizeIdlist;
```
Comment (AndriyLymych) [inline src/components/forms/model-form/model-form.js:88]: use setFieldValue

File: src/components/forms/model-form/model-form.js [hunk 3]
```diff
@@ -129,42 +123,43 @@ const ModelForm = ({ model, id, isEdit }) => {
...
         setModelImage(event.target.result);
       };
       reader.readAsDataURL(e.target.files[0]);
       setUpload(e.target.files[0]);
     }
   };
-  const handleConstructor=()=>{
+  const handleConstructor = () => {
     dispatch(push(`/constructor/${id}`));
-  }
```
Comment (AndriyLymych) [inline src/components/forms/model-form/model-form.js:163]: dont use magic strings

File: src/components/forms/model-form/model-form.js (hunk 2 above)
Comment (Haytham1337) [reply src/components/forms/model-form/model-form.js]: used setFieldValue

File: src/components/forms/model-form/model-form.js (hunk 3 above)
Comment (Haytham1337) [reply src/components/forms/model-form/model-form.js:163]: Replace string for constant

**Review by AndriyLymych — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Review by M1ndForce — APPROVED** [review]

**Pull request merged**